src/createServer.js

```diff
@@ -1,3 +1,32 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { errorValidation } = require('./errorValidation');
+const { responce } = require('./responce');
+const { getURL } = require('./getURL');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const splitUrl = req.url.split('?');
+    const { textToConvert, toCase } = getURL(splitUrl);
+    const checkError = errorValidation(textToConvert, toCase);
```

Comment:
It's an array, not a boolean type
```suggestion
    const errorMessages = errorValidation(textToConvert, toCase);
```